### PR TITLE
Final clean up of access node codebase

### PR DESCRIPTION
**Pull request by szegedim**
This is a final cleanup of access node codebase. The reason leaving to the end is to be able to PR and merge separately allowing flexibility in case of occasional rollbacks. It compares to the main observer/follower refactoring PR right now.

### CODE REVIEW

File: cmd/access/node_builder/access_node_builder.go [hunk 1]
```diff
@@ -410,6 +380,9 @@ func (builder *FlowAccessNodeBuilder) ParseFlags() error {
 func (builder *FlowAccessNodeBuilder) extraFlags() {
 	builder.ExtraFlags(func(flags *pflag.FlagSet) {
 		defaultConfig := DefaultAccessNodeConfig()
+		var dummyBool bool
+		var dummyString string
+		var dummyStringSlice []string
```
Comment (smnzhu) [inline cmd/access/node_builder/access_node_builder.go:385]: let's just remove all deprecated CLI args

**Review by smnzhu — APPROVED** [review]

File: cmd/access/node_builder/access_node_builder.go (hunk 1 above)
Comment (szegedim) [reply cmd/access/node_builder/access_node_builder.go:385]: Done.

Comment (szegedim) [reply cmd/access/node_builder/access_node_builder.go:385]: Fixed: https://github.com/GetElastech/flow-go/commit/33623bb268ecfaedd1680372860ab98e6a35b7b6

**Comment by szegedim** [comment]
Replaced by: https://github.com/onflow/flow-go/pull/2554

**Pull request closed without merging**